src/processSection.js

```diff
@@ -58,8 +58,12 @@ module.exports = function processSection(markdown, o) {
         after: after // encodeURI(after)
       })
       .done(function onComplete(result, success, xhr) {
-        // we should need fewer things here:
-        o.onComplete(xhr.status, after, html, _el, uniqueId, __form, o);
+        if (result == "false") {
```

Comment:
> .done(): Description: Add handlers to be called when the Deferred object is resolved.

How about if-ing the `xhr.status` to all the unwanted values like 404, 500, etc. here?